Update Program.cs

mdragt opened the pull request:
Add display the latest results and updated the isUpdateModel function in order to compare the new training results with the prior scores. If the results aren't better, the blob with the new training results will be deleted.

mdragt committed:
Update Program.cs

Add display the latest results and updated the isUpdateModel function in order to compare the new training results with the prior scores. If the results aren't better, the blob with the new training results will be deleted.

jmarymee commented:
LOL – I am behind. Will review this evening ☺
JD

From: Marian Dragt [mailto:notifications@github.com]
Sent: Tuesday, March 15, 2016 2:45 AM
To: jmarymee/convirga convirga@noreply.github.com
Subject: [convirga] Update Program.cs (#3)

Add display the latest results and updated the isUpdateModel function in order to compare the new training results with the prior scores. If the results aren't better, the blob with the new training results will be deleted.

---

You can view, comment on, or merge this pull request online at:

  https://github.com/jmarymee/convirga/pull/3

Commit Summary
-   Update Program.cs

File Changes
-   M AMLRetrainUsingSQL/Program.cshttps://github.com/jmarymee/convirga/pull/3/files#diff-0 (63)

Patch Links:
-   https://github.com/jmarymee/convirga/pull/3.patch
-   https://github.com/jmarymee/convirga/pull/3.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/jmarymee/convirga/pull/3

mdragt commented:
Haha no prob! Let me know if you have any questions.
Best, Marian

From:  JD Marymee notifications@github.com
Reply-To:  jmarymee/convirga reply@reply.github.com
Date:  Monday 21 March 2016 at 22:31
To:  jmarymee/convirga convirga@noreply.github.com
Cc:  Marian Dragt info@md2c.nl
Subject:  Re: [convirga] Update Program.cs (#3)

LOL – I am behind. Will review this evening ☺ 
JD 

From: Marian Dragt [mailto:notifications@github.com] 
Sent: Tuesday, March 15, 2016 2:45 AM 
To: jmarymee/convirga convirga@noreply.github.com 
Subject: [convirga] Update Program.cs (#3) 

Add display the latest results and updated the isUpdateModel function in order to compare the new training results with the prior scores. If the results aren't better, the blob with the new training results will be deleted. 

---

You can view, comment on, or merge this pull request online at: 

 https://github.com/jmarymee/convirga/pull/3 

Commit Summary 
-   Update Program.cs 

File Changes 
-   M AMLRetrainUsingSQL/Program.cshttps://github.com/jmarymee/convirga/pull/3/files#diff-0 (63) 

Patch Links: 
-   https://github.com/jmarymee/convirga/pull/3.patch 
-   https://github.com/jmarymee/convirga/pull/3.diff 

— 
You are receiving this because you are subscribed to this thread. 
Reply to this email directly or view it on GitHub: 
https://github.com/jmarymee/convirga/pull/3 

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub

mdragt commented:
Hi JD, did you have time to review my pull requests? Best, MD

From: JD Marymee [mailto:notifications@github.com] 
Sent: Monday, March 21, 2016 10:32 PM
To: jmarymee/convirga convirga@noreply.github.com
Cc: Marian Dragt info@md2c.nl
Subject: Re: [convirga] Update Program.cs (#3)

LOL – I am behind. Will review this evening ☺ 
JD 

From: Marian Dragt [mailto:notifications@github.com] 
Sent: Tuesday, March 15, 2016 2:45 AM 
To: jmarymee/convirga <convirga@noreply.github.com <mailto:convirga@noreply.github.com> > 
Subject: [convirga] Update Program.cs (#3) 

Add display the latest results and updated the isUpdateModel function in order to compare the new training results with the prior scores. If the results aren't better, the blob with the new training results will be deleted. 

---

You can view, comment on, or merge this pull request online at: 

https://github.com/jmarymee/convirga/pull/3 

Commit Summary 
- Update Program.cs 

File Changes 
- M AMLRetrainUsingSQL/Program.cshttps://github.com/jmarymee/convirga/pull/3/files#diff-0 (63) 

Patch Links: 
- https://github.com/jmarymee/convirga/pull/3.patch 
- https://github.com/jmarymee/convirga/pull/3.diff 

— 
You are receiving this because you are subscribed to this thread. 
Reply to this email directly or view it on GitHub: 
https://github.com/jmarymee/convirga/pull/3 

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub https://github.com/jmarymee/convirga/pull/3#issuecomment-199495386   https://github.com/notifications/beacon/AC-SyiVZYFV9hRsSINRFhSRWuOF8W4M1ks5pvw5JgaJpZM4Hw2b-.gif 

Geen virus gevonden in dit bericht.
Gecontroleerd door AVG - www.avg.com http://www.avg.com 
Versie: 2016.0.7497 / Virusdatabase: 4545/11918 - datum van uitgifte: 03/30/16

mdragt committed:
Store retraining results

Add StoreRetrainedResults() to store retraining results and model for future analysis
